README.md

```diff
@@ -1,61 +1,94 @@
+  which is described below.
 
-  The DMRG module is built from 3 source files and 5 header files:
+## Code Structure
 
-     cDMRG.cc -- main block
-     cio.cc -- input/output
-     readvec.cc -- reading vectors
-     cSite.h -- site set
-     cDMRGeps.h -- DMRG sweeps
-     localmpoproj.h -- Hamiltonian MPOs
-     cio.h -- input/output header
-     readvec.h -- readvec header
+cDMRG consists of two main modules:
+- DMRG Module using the ITensor library 
+- Input Module written in Mathematica
+
+### DMRG Module
 
-  First install the C++ version of ITensor: https://itensor.org/. Change the address in the make file "make_cDMRG" to point to the built ITensor library. Then run the make file from command line. It should produce an executable file named "cDMRG".
+The DMRG module consists of the following files:
 
+- `cDMRG.cc` - main block
+- `cio.h`, `cio.cc` - input/output
+- `readvec.h`, `readvec.cc` - reading vectors
+- `cSite.h` - site set
+- `cDMRGeps.h` - DMRG sweeps
+- `localmpoproj.h` - Hamiltonian MPOs
 
 ### Input Module
 
-  The DMRG module is called by the Mathematica package "cDMRG_input.m", which generates the necessary inputs (local basis and operators) for DMRG. A separate documentation of "cDMRG_input.m" is provided in the notebook "cDMRG_input_documentation.nb". The package can be modified by changing the notebook "cDMRG_input.nb" and saving it as a package (.m).
+The DMRG module is called by the Mathematica package `cDMRG_input.m`, which generates the necessary
+inputs (local basis and operators) for DMRG.
+A separate documentation of `cDMRG_input.m` is provided in the notebook `cDMRG_input_documentation.nb`.
+The package can be modified by changing the notebook `cDMRG_input.nb` and saving it as a package `.m`.
 
-  One calls the program from command line as 
+## How to Use
 
-     math -noprompt -run '<<cDMRG_input.m' N gamma Nwell V0 M basisid epsid saveid &
-  
-  where
-  
-     N -- number of particles
-     gamma -- dimensionless interaction strength
-     Nwell -- number of potential minima
-     V0 -- potential depth in units of recoil
-     M -- number of segments
-     basisid -- ID for basis parameters
-     epsid -- ID for DMRG parameters
-     saveid -- ID for save parameters.
+Call the program from the command line as 
+```consol
+math -noprompt -run '<<cDMRG_input.m' N gamma Nwell V0 M basisid epsid saveid &
+```
 
-  The results are stored in the directory "Runs".
+where
 
+- `N` - `int` number of particles
+- `gamma` - `float` dimensionless interaction strength
+- `Nwell` - `int` number of potential minima
+- `V0` - `float` potential depth in units of recoil
+- `M` - `int` number of segments
+- `basisid` - `int` ID for basis parameters
+- `epsid` - `int` ID for DMRG parameters
+- `saveid` - `int` ID for save parameters
 
-## IDs / Stored Inputs
+For instance a short calculation can be initiated by
+```consol
+math -noprompt -run '<<cDMRG_input.m' 2 0.1 4 1.0 20 1 1 1 &
```

Comment:
Perhaps change it to  4 10 4 1 8 1 1 1 &
The stored results in "Runs" correspond to these parameters. Takes 1-2 min to run.
Or, I'd change the number of segments from 20 to 10 or even 5.